courses/free-courses-hi.md

```diff
@@ -184,6 +184,7 @@
 ### DevOps
 
 * [DevOps Training Videos in Hindi](https://www.youtube.com/playlist?list=PLQbQOmlGYH3sxlq9ugoq1ipNFP7tus5Gd) - edureka! Hindi
+* [Devops Tutorial For Beginners In Hindi | DevOps Tutorial | DevOps Full Course](https://youtu.be/tgmM3_2dZwg) - Great Learning
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes 
No shortened links